### PR TITLE
Separate workflows

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
* Supersedes #921
* Removes outdated & unused[^1] `checkbot.yml` superseded by [iterative/cml-playground](https://github.com/iterative/cml-playground)

[^1]: Has to be manually triggered; last run was 5 months ago, and it failed. 🙈

### CODE REVIEW

**Review by dacbd — APPROVED** [review]
lgtm

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git config --global user.name Olivaw[bot]
+          git config --global user.email 64868532+iterative-olivaw@users.noreply.github.com
+          git checkout -b bump/$(npm version ${{ github.event.inputs.bump }})
+          git push --set-upstream origin --follow-tags HEAD
+          gh pr create --title "Bump version to $(git describe --tags)" --fill
```
Comment (casperdcl) [inline .github/workflows/release.yml:24]: cml ci && cml pr?

Comment (0x2b3bfa0) [reply .github/workflows/release.yml:24]: `:auryn:`?

Comment (casperdcl) [reply .github/workflows/release.yml:24]: :dogfood:

Comment (0x2b3bfa0) [reply .github/workflows/release.yml:24]: @casperdcl, feel free to open a follow-up issue or pull request proposing using `cml` as part of the `cml` relese project, but it seems the perfect recipe for a what-could-possibly-go-wrong–level failure in our release pipeline when we need it most.

Comment (casperdcl) [reply .github/workflows/release.yml:24]: #1165 :)

**Pull request merged**